packages/suite/src/types/suite/guide.tsx

```diff
@@ -9,7 +9,7 @@ export type FeedbackView = 'FEEDBACK_TYPE_SELECTION' | 'FEEDBACK_BUG' | 'FEEDBAC
 
 export type ActiveView = GuideView | FeedbackView;
 
-export type Category =
+export type SuiteCategory =
```

Comment:
optional nit: `SuiteCategory` is IMO quite ambiguous. We tend to put the `Suite` prefix, suffix or infix in a lot of names and it kinda lost its meaning already.

I think every symbol should be called with context in mind and having a `Suite` in the name in context of a Suite app is superfluous. What about we call this eg. `FeedbackCategory` or `BugCategory`? .)

Reply:
db06e246a139a0cf36b922c5f1f96d5596904e87